Project/Engine/src/sysFrame/System/Log/Logger.cpp

```
@@ -39,23 +38,23 @@ void Log::Initialize() {
+    spdlog::set_pattern("[%D-%R:%S.%e%f%F][Thread:%t][%l]:%v");
 
     spdlog::info("LogSystem Enabled");
 }
 
-void Log::Info(const std::string& msg) const {
+void Logger::Info(const std::string& msg) const {
     combined_logger->info(msg);
 }
 
-void Log::Debug(const std::string& msg) const {
+void Logger::Debug(const std::string& msg) const {
```

cppcheck (2.10-2)

[style] The function 'Debug' is never used.

(unusedFunction)